sgx-tools/gen-token.go

```diff
@@ -32,6 +32,11 @@ For example, generate the token file according to the given signature file:
 		},
 	},
 	Action: func(context *cli.Context) error {
+
+		if intelsgx.IsSGXLaunchControlSupported() {
```

Comment:
So you did well here with a runtime detection but why applying a static form in skeleton?